File: tests/fixtures/sample_code/sample.c
// Sample C file for testing

#include <stdio.h>

/**
 * Greet a person by name
 */
char* greet(char* name) {
    static char buffer[100];
    sprintf(buffer, "Hello, %s!", name);
    return buffer;
}

/**
 * Add two numbers
 */
int add(int a, int b) {
    return a + b;
}

void no_args() {
    printf("No arguments\n");
}

File: tests/fixtures/sample_code/sample.go
// Sample Go file for testing

package main

import "fmt"

// Greet greets a person by name
func Greet(name string) string {
	return fmt.Sprintf("Hello, %s!", name)
}

// Add adds two numbers
func Add(a, b int) int {
	return a + b
}

func noArgs() {
	fmt.Println("No arguments")
}
